packages/ui/src/components/Button/Button.tsx

```diff
@@ -44,6 +44,7 @@ export function Button({
                 disabled:bg-white disabled:border-transparent disabled:text-gray-400
                 `,
                 Boolean(Icon) && Boolean(children) && `gap-2`,
+                Boolean(Icon) && !children && `px-4 py-3`,
```

Comment:
The button is still not squared, but now in another way :D

<img width="365" alt="image" src="https://github.com/prezly/theme-kit-js/assets/1641218/3340bd31-1abd-4597-bd3c-85663545b753">

Reply:
Fixed it. https://github.com/prezly/theme-kit-js/pull/404/commits/4f07d07b37ab7c8da93e1458eb4a0871d3147dd7